.babelrc

```diff
@@ -1,4 +1,6 @@
+
+popo
 {
-  "plugins": ["transform-class-constructor-call"],
-  "presets": ["es2015", "stage-2"]
+  "presets": ["es2015", "stage-2"],
```

Comment:
I recently added what "feels" like a minimal Babel/Jest/Yarn setup in our [Auth0 sync code](https://github.com/globality-corp/globality-auth0). Might be useful to standardize on.